lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivity.tsx

```diff
@@ -45,9 +45,12 @@ export default function OrganizationActivity({
           defaultOrderField="title"
           renderItem={stats => (
             <RouterLink href={urlPath`/courses/${String(stats.id)}`} asChild>
-              <Link>{stats.title}</Link>
+              <Link underline="always" variant="text">
+                {stats.title}
+              </Link>
             </RouterLink>
           )}
+          navigateTo={stats => urlPath`/courses/${String(stats.id)}`}
```

Comment:
> One way this could work would be if the `onConfirm` handler just found the link in the selected row and clicked it.

Hm, I don't know. That would mean that either we pass some refs to the link from concrete activity sections to the `OrderableActivityTable`, or `OrderableActivityTable` tries to search for it via query selector.

The first option is not very practical, and the second couples `OrderableActivityTable` with a specific structure (what if there are no links? What if there are multiple links?).

I would propose defining a local helper function in concrete activity sections which ensures we don't leave incorrect links. Something in the lines of ``const buildCourseURL = (id: number) => urlPath`/courses/${String(id)}` `` and then we call that for the actual link and fir this prop.